packages/itwin/property-grid/tsconfig.json

```diff
@@ -8,5 +8,5 @@
     "resolveJsonModule": true,
     "lib": ["DOM", "ES2019"]
   },
-  "include": ["src"],
+  "include": ["src", "playwright.config.ts"]
```

Comment:
this will build and include build output of `playwright.config.ts` into the `lib`, won't it?

Reply:
Removed